packages/ses/src/tame-locale-methods.js

```diff
@@ -39,7 +39,7 @@ const nonLocaleCompare = tamedMethods.localeCompare;
 
 export default function tameLocaleMethods(intrinsics, localeTaming = 'safe') {
   if (localeTaming !== 'safe' && localeTaming !== 'unsafe') {
-    throw new Error(`unrecognized dateTaming ${localeTaming}`);
+    throw new TypeError(`unrecognized dateTaming ${localeTaming}`);
```

Comment:
```suggestion
    throw new TypeError(`unrecognized localeTaming ${localeTaming}`);
```
I'm sure this was my mistake, but is a good time to fix it.